mtp_api/apps/security/migrations/0034_auto_20210114_1234.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.2.17 on 2021-01-14 12:34
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('security', '0033_auto_20210111_1709'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='check',
+            name='auto_accept_rule',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='checks', to='security.CheckAutoAcceptRule'),
+        ),
+        migrations.AlterField(
+            model_name='check',
+            name='rejection_reasons',
+            field=django.contrib.postgres.fields.jsonb.JSONField(default=dict),
+        ),
```

Comment:
Didn't the `Check` model have a `rejection_reasons` field before?

Maybe it's a leftover from another migration/branch?